packages/alfa-rules/src/common/predicate/has-cascaded-value-declared-in-inline-style.ts

```diff
@@ -1,8 +1,10 @@
 import { Element } from "@siteimprove/alfa-dom";
```

Comment:
Iirc, this predicate can now be deleted since it was only used in these 3 rules.
(plus remove from `tsconfig` and `predicate.ts`)

Reply:
That's right, I am deleting it